pkg/kubernetes/scripts/kube-client.js

```diff
@@ -38,6 +38,7 @@
 
     var KUBE = "/api/v1";
     var OPENSHIFT = "/oapi/v1";
+    var KUBEVIRT = "/apis/kubevirt.io/v1alpha1";
```

Comment:
Is there a way we can discover this instead having to hard code it?

Reply:
I guess prefix`/apis/kubevirt.io` is ok. `v1alpha1` could be in theory extracted from response to `GET /apis/kubevirt.io` as value of `preferredVersion` field.

But that's probably not what we want. All the kubevirt related code relies on `v1alpha1` and is not guarantied to work with other versions (since there is currently no other version). I case of api update (and related api prefix update) we would need most probably update the code as well.

Reply:
I've seen a lot of use cases where users use a new version of this (usually the container) with an older kubernetes version. So if we hard code this then in the future when we bump to ```v1``` this code will no longer be usable with older versions. 

But it's also possible that the API will change so much that we'll never be able to support any other version in the same js code base. So I guess it makes sense to leave it like this for now and we can make the decision about continuing to support ```v1alpha1``` once we know what the differences are.